modules/references/client/components/create-reference/Navigation.js

```diff
@@ -11,6 +11,7 @@ export function Navigation({ t, disabled, tab, tabs, tabDone, onBack, onNext, on
   const backButton = (
     <button
       type="button"
+      role="button"
```

Comment:
I think `<button>` elements might already be getting button role by default? 

See ["Permitted ARIA roles" list](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button).

Reply:
Yes, you're right, let me see if it finds it without it explicitly set. I couldn't see where it said that in your link, but in this one it says it clearly: https://www.w3.org/TR/2014/REC-html5-20141028/forms.html#the-button-element

Reply:
Yup, still works without that, so I removed that, thanks!